### PR TITLE
Created a new segment to track Air Quality "Air"

### PR DESCRIPTION
**Pull request by gavink97**
Air allows users to track air quality in a give location via the Open Weather API.

It utilizes US AQI which is the US standard to measure Air quality and returns different colors + symbols for each index level.

Users can use the TMUX_POWERLINE_SEG_AIR_COLOR variable to have the segment change colors as the Air Quality goes between different index levels.

### CODE REVIEW

File: lib/headers.sh [hunk 1]
```diff
@@ -19,3 +19,5 @@ source "${TMUX_POWERLINE_DIR_LIB}/formatting.sh"
 source "${TMUX_POWERLINE_DIR_LIB}/muting.sh"
 source "${TMUX_POWERLINE_DIR_LIB}/powerline.sh"
 source "${TMUX_POWERLINE_DIR_LIB}/rcfile.sh"
+
```
Comment (erikw) [inline lib/headers.sh:22]: Segments should not be sourced here. Please remove and verify that the segment still works :).

**Review by erikw — CHANGES_REQUESTED** [review]
Wow cool! Just ones thing to fix

**Comment by gavink97** [comment]
> Wow cool! Just ones thing to fix

Okay I fixed it 👍🏼

**Review by erikw — APPROVED** [review]

**Comment by erikw** [comment]
Thx

**Pull request merged**